src/Compilers/CSharp/Portable/Symbols/TypeSymbolWithAnnotations.cs

```diff
@@ -11,6 +11,28 @@
 
 namespace Microsoft.CodeAnalysis.CSharp.Symbols
 {
+    internal enum NullableAnnotation : byte
+    {
+        Unknown,     // No information. Think oblivious.
```

Comment:
>Unknown [](start = 8, length = 7)

Please consider adding explicit `= 0` for this value.

Reply:
> Please consider adding explicit = 0 for this value.

What is so special about this enum type? I doesn't look like we do this in other enums

---
In reply to: [231624781](https://github.com/dotnet/roslyn/pull/30913#discussion_r231624781) [](ancestors = 231624781)

Reply:
Then it's clear that `Unknown` is the default value, especially since the enum is used in `struct` types.

---
In reply to: [231625551](https://github.com/dotnet/roslyn/pull/30913#discussion_r231625551) [](ancestors = 231625551,231624781)

Reply:
> Then it's clear that Unknown is the default value, especially since the enum is used in struct types.

Why it is clear without for other enum types? It is easy enough for me to make the change, however I do not see anything special about this particular type. So I am not sure what kind of precedent we are about to create here.

---
In reply to: [231626586](https://github.com/dotnet/roslyn/pull/30913#discussion_r231626586) [](ancestors = 231626586,231625551,231624781)